cmake/modules/AddPureSwift.cmake

```diff
@@ -269,7 +269,8 @@ function(add_pure_swift_host_tool name)
 
   # Option handling
   set(options)
-  set(single_parameter_options)
+  set(single_parameter_options
+    SWIFT_COMPONENT)
```

Comment:
Could add the precondition that `add_swift_host_tool` has as well. But LGTM otherwise!

Reply:
We don't have that precondition in other places this is replicated.

Reply:
🤷 Just makes it more obvious when someone forgets to do it. We can add it in a later PR.